plugins/woocommerce/tests/e2e-pw/tests/activate-and-setup/task-list.spec.js

```diff
@@ -1,19 +1,61 @@
-const { test, expect } = require( '@playwright/test' );
-const { tags } = require( '../../fixtures/fixtures' );
+const { tags, test, expect } = require( '../../fixtures/fixtures' );
 const { ADMIN_STATE_PATH } = require( '../../playwright.config' );
 
+const get_task_list_state = async ( wcAdminApi ) => {
+	const {
+		statusText,
+		data: { woocommerce_task_list_hidden },
+	} = await wcAdminApi.get( 'options?options=woocommerce_task_list_hidden' );
```

Comment:
I tried using a relative path (`./options?options=woocommerce_task_list_hidden`) to make compatible with subdir multisites in the future, but wcAdminApi threw a `404` response :man_shrugging:. So I didn't attach a leading `./` in the path for now.

Reply:
There's no need to add a leading ./ here, because there is the `wc-admin` part that's added before that by the `wcAdminApi`. It makes sense for a url like `site_url/wc-admin/./options?` to fail

Reply:
Got it. Thanks a lot for clarifying and for reviewing! Merging this now.